tfx_bsl/beam/run_inference.py

```diff
@@ -71,27 +73,26 @@
                                classification_pb2.Classifications]]
 
 
+class OperationType(object):
```

Comment:
Inherit from Enum?

Reply:
I was concerned about the lack of `Enum` on Python 2.7, but we already have access to `enum34` (backported Enum), since this is a dependency of Beam. If you are fine with marking `enum34` as tfx_bsl`s dependency, I think I can use it.

Reply:
Ah, good point. enum34 is not a necessary dependency to add. Let's keep it this way.

Reply:
Let's have a TODO to convert this into enum once we stopped supporting python 2.7, which should be sometime Q2-Q3 this year.

Reply:
Done, a TODO has been added.